### ISSUE TITLE
Should be able to scroll filter column selection

### ISSUE THREAD

**Issue by midgleyc** [prisma/studio#627]
If I have more than 33 columns, I can't select the later ones when trying to filter by column.

Firefox keeps the filter box where it is, but the column list goes off the bottom of the screen. Chrome bumps the filter box up so I can see more entries, but there are still some that go off the screen.

"Fields" shows all the entries (for me) but might have the same problem with a truly huge number of columns.

Prisma version (`prisma -v` or `npx prisma -v`):

```
prisma               : 2.16.1
@prisma/client       : Not found
Current platform     : debian-openssl-1.1.x
Query Engine         : query-engine 8b74ad57aaf2cc6c155f382a18a8e3ba95aceb03 (at node_modules/@prisma/engines/query-engine-debian-openssl-1.1.x)
Migration Engine     : migration-engine-cli 8b74ad57aaf2cc6c155f382a18a8e3ba95aceb03 (at node_modules/@prisma/engines/migration-engine-debian-openssl-1.1.x)
Introspection Engine : introspection-core 8b74ad57aaf2cc6c155f382a18a8e3ba95aceb03 (at node_modules/@prisma/engines/introspection-engine-debian-openssl-1.1.x)
Format Binary        : prisma-fmt 8b74ad57aaf2cc6c155f382a18a8e3ba95aceb03 (at node_modules/@prisma/engines/prisma-fmt-debian-openssl-1.1.x)
Studio               : 0.346.0
```

**Comment by madebysid** [comment]
Ah, a classic frontend problem. I'll fix this as soon as I can!

**Comment by madebysid** [comment]
Hey @midgleyc This is now fixed in Prisma CLI `2.18.0-dev.xx`. If this is blocking, you should be able to `npm install prisma@dev` to get this immediately. However, I'd recommend updating to the next stable version when it comes out, since `dev` versions are mostly for internal use and may contain unannounced breaking changes.

This will also make it to stable v2.18.0 scheduled for this Tuesday!
